tests/tests_report_workflow/test_test_report_runner_dashboards.py

```diff
@@ -51,14 +51,29 @@ def test_generate_file(self, report_args_mock: MagicMock) -> None:
         report_args_mock.test_type = "integ-test"
         report_args_mock.release_candidate = "100"
 
-        test_run_runner = TestReportRunner(report_args_mock, self.TEST_MANIFEST_OPENSEARCH_DASHBOARDS)
-        test_run_runner_data = test_run_runner.update_data()
+        test_report_runner = TestReportRunner(report_args_mock, self.TEST_MANIFEST_OPENSEARCH_DASHBOARDS)
+        test_report_runner_data = test_report_runner.update_data()
 
         with TemporaryDirectory() as path:
             output_path = os.path.join(path.name, "test-report.yml")
-            test_run_runner.generate_report(test_run_runner_data, path.name)
+            test_report_runner.generate_report(test_report_runner_data, path.name)
             self.assertTrue(os.path.isfile(output_path))
 
+    @patch("report_workflow.report_args.ReportArgs")
+    def test_ci_group(self, report_args_mock: MagicMock) -> None:
+        report_args_mock.test_manifest_path = self.TEST_MANIFEST_OPENSEARCH_DASHBOARDS_PATH
+        report_args_mock.artifact_paths = {"opensearch-dashboards": self.DATA_DIR}
+        report_args_mock.test_run_id = 123
+        report_args_mock.base_path = self.DATA_DIR
+        report_args_mock.test_type = "integ-test"
+        report_args_mock.release_candidate = "100"
+
+        test_report_runner = TestReportRunner(report_args_mock, self.TEST_MANIFEST_OPENSEARCH_DASHBOARDS)
+        test_report_runner_data = test_report_runner.update_data()
+
+        self.assertEqual(len(test_report_runner_data["components"]), 9)
+        self.assertEqual(test_report_runner_data["components"][0]["name"], "OpenSearch-Dashboards-ci-group-1")
```

Comment:
Can you also add assertions for all ci-groups? Until 9?

Reply:
Already assert the total number of components in the report manifest though. I would add assertions for all of them.